### PR TITLE
crystal: add French translation, update page

### PR DESCRIPTION
**Pull request by noraj**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.9.2

### CODE REVIEW

File: pages/common/crystal.md [hunk 1]
```diff
@@ -11,6 +11,18 @@
 
 `crystal build {{path/to/file.cr}}`
 
+- Read Crystal source code from command line or stdin, and execute it:
```
Comment (kbdharun) [inline pages/common/crystal.md:14]:
```suggestion
- Read Crystal source code from the command line or `stdin`, and execute it:
```

File: pages/common/crystal.md [hunk 2]
```diff
@@ -11,6 +11,18 @@
...
+
+`crystal eval '{{code}}'`
+
+- Generate API documentation from inline docstrings in Crystal files:
+
+`crystal docs`
+
+- Compile and run a Crystal spec suite:
```
Comment (kbdharun) [inline pages/common/crystal.md:22]:
```suggestion
- Compile and run a Crystal specification suite:
```
Maybe we can expand it.

File: pages.fr/common/crystal.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Exécute un fichier Crystal :
+
+`crystal {{chemin/vers/fichier.cr}}`
+
+- Compile un fichier et toutes ses dépendances en un seul exécutable :
+
+`crystal build {{chemin/vers/fichier.cr}}`
+
+- Lit le code source Crystal à partir de la ligne de commande ou de stdin, et l'exécute :
```
Comment (kbdharun) [inline pages.fr/common/crystal.md:14]:
```suggestion
- Lit le code source Crystal à partir de la ligne de commande ou de `stdin`, et l'exécute :
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**